daphne_server/examples/service.rs

```diff
@@ -0,0 +1,82 @@
+use std::path::PathBuf;
+
```

Comment:
It's not very clear what this is an example of. More documentation is needed.

Minimally, add a docucomment describing what this does (or will do, once it's complete).

Reply:
This is actually the test binary. I intend to write a readme for this later on